common/app/conf/switches.scala

```diff
@@ -471,6 +471,11 @@ object Switches {
     safeState = Off, sellByDate = new LocalDate(2015, 5, 11)
   )
 
+  val ABMtTopBelowNav = Switch("A/B Tests", "ab-mt-top-below-nav",
```

Comment:
It looks like it's also being used as part of the test framework here: https://github.com/guardian/frontend/blob/master/static/src/javascripts/projects/common/modules/experiments/ab.js#L82